ldcpy/util.py

```diff
@@ -54,31 +53,29 @@ def print_stats(ds, varname, set1, set2, time=0, sig_dig=4):
     """
     Print error summary statistics of two DataArrays
 
-    Parameters:
-    ===========
-    ds -- xarray.Dataset
-        an xarray dataset containing multiple netCDF files concatenated across a 'collection' dimension
-    varname -- string
-        the variable of interest in the dataset
-    set1 -- string
-        the collection label of the "control" data
-    set2 -- string
-        the collection label of the (1st) data to compare
-
-    Keyword Arguments:
-    ==================
-    time -- int
-        the time index used to compare the two netCDF files (default 0)
+    Parameters
+    ==========
+    ds : xarray.Dataset
+        An xarray dataset containing multiple netCDF files concatenated across a 'collection' dimension
+    varname : str
+        The variable of interest in the dataset
+    set1 : str
+        The collection label of the "control" data
+    set2 : str
+        The collection label of the (1st) data to compare
+    time : int, optional
+        The time index used to compare the two netCDF files (default 0)
+
+    sig_dig : int, optional
+        (default 4)
```

Comment:
I wasn't sure what the description for `sig_dig` should be. What does `sig_dig` mean? ;)

Reply:
Oops, that should be the number of significant digits to print for each metric, but it's actually one less than that. I just pushed a change to fix the problem. Thanks!

Reply:
Great... Just merged latest changes from master into this PR. Feel free to merge this into master once the CI is passing.